rest_witchcraft/serializers.py

```diff
@@ -45,6 +46,7 @@ def build_standard_field_kwargs(self, field_name, field_class, column_info):
         Analyze model column to generate field kwargs.
         """
         field_kwargs = column_info.field_kwargs
+        field_kwargs["label"] = capfirst(" ".join(field_name.split("_")).strip())
```

Comment:
this will affect all serializers. good idea?

Reply:
Sure